lntest/itest/lnd_multi-hop_htlc_local_chain_claim_test.go

```diff
@@ -209,90 +178,59 @@ func testMultiHopHtlcLocalChainClaim(net *lntest.NetworkHarness, t *harnessTest,
 	// At this point we suspend Alice to make sure she'll handle the
 	// on-chain settle after a restart.
 	restartAlice, err := net.SuspendNode(alice)
-	if err != nil {
-		t.Fatalf("unable to suspend alice: %v", err)
-	}
+	require.NoError(t.t, err)
 
 	// Mine a block to confirm the two transactions (+ the coinbase).
 	block = mineBlocks(t, net, 1, expectedTxes)[0]
-	if len(block.Transactions) != expectedTxes+1 {
-		t.Fatalf("expected 3 transactions in block, got %v",
-			len(block.Transactions))
-	}
+	require.Equal(t.t, expectedTxes+1, len(block.Transactions))
 
 	// Keep track of the second level tx maturity.
 	carolSecondLevelCSV := uint32(defaultCSV)
 
 	// When Bob notices Carol's second level transaction in the block, he
 	// will extract the preimage and broadcast a second level tx to claim
 	// the HTLC in his (already closed) channel with Alice.
-	bobSecondLvlTx, err := waitForTxInMempool(net.Miner.Node,
-		minerMempoolTimeout)
-	if err != nil {
-		t.Fatalf("transactions not found in mempool: %v", err)
-	}
+	bobSecondLvlTx, err := waitForTxInMempool(
```

Comment:
I wonder if it'd make sense to `require.NoError` inside the `waitForTxInMempool` func?

Reply:
Yes, it probably would make sense to do so. But this function is used in so many places that refactoring that is probably its own PR.